src/plugins/xfconf/README.md

```diff
@@ -2,9 +2,9 @@
 - infos/author = Richard Stöckl <e11908080@student.tuwien.ac.at>
 - infos/licence = BSD
 - infos/needs =
-- infos/provides = storage/xfconf
+- infos/provides = storage/xml
```

Comment:
This should be in a different PR to merge it first.

Reply:
This was actually an accident. I committed it in two different PRs, the other one was #4947. Since I rebased this PR with the master, this PR will not change anything regarding this.